### PR TITLE
feat(22966): replace old integrator and sdk usages

### PR DESCRIPTION
**Pull request by nnorbert**
Replace old SDK usages

### CODE REVIEW

File: apps/dashboard/utils/authentication-factory.ts [hunk 1]
```diff
@@ -243,8 +244,11 @@ class SDKFactory {
         featureFlags: {
           baseUrl: this.featureFlagApiHostUrl,
         },
+        journey: {
+          baseUrl: this.featureFlagApiHostUrl,
+        }
       },
-    })
+    }) as BuildingBlocksSDK
```
Comment (Thurfjell) [inline apps/dashboard/utils/authentication-factory.ts:251]: Suggestion: Remove type assertion and set the type (probably like Pick services from returntype of getbuildingblockssdk or something).

Comment (nnorbert) [reply apps/dashboard/utils/authentication-factory.ts]: Done!

**Review by Thurfjell — APPROVED** [review]

File: apps/journey-api/.env.sample
Comment (peschina) [inline apps/journey-api/.env.sample:1]: We will need to remove these environment variables in dev too

File: apps/payments-api/.env.sample
Comment (peschina) [inline apps/payments-api/.env.sample:1]: Same here, needs to be added in dev

**Review by peschina — APPROVED** [review]
LGTM. We need to update env vars in dev before merging

**Pull request merged**